### ISSUE TITLE
Hardcoded downsampling target leads to error

### ISSUE THREAD

**Issue by ftilmann** [gesape/AutoStatsQ#28]
In the restitution step when applying to LH data (deltaT=1 s) with the following parameters
```
- !autostatsq.config.RestDownRotConfig
  rest_data: true
  freqlim: [0.005, 0.01, 0.2, 0.25]
  rotate_data: true
  deltat_down: 2.0
```

I get an error about Unavailable Decimations Rate:
```
INFO:Restitution: Starting restitution of data.
INFO:Restitution: Number of responses found: 1.
INFO:Restitution: Starting restitution of data, shallow subset.
Traceback (most recent call last):
  File "/home/tilmann/.local/bin/autostatsq", line 11, in <module>
    load_entry_point('autostatsq==0.1', 'console_scripts', 'autostatsq')()
  File "/home/tilmann/.local/lib/python3.8/site-packages/autostatsq-0.1-py3.8.egg/autostatsq/network_control.py", line 978, in main
    tr.downsample_to(0.1)
  File "/usr/local/lib/python3.8/dist-packages/pyrocko/trace.py", line 663, in downsample_to
    raise util.UnavailableDecimation('ratio = %g' % ratio)
pyrocko.util.UnavailableDecimation: ratio = 0.1
```
Looking t the code (main branch) it seems the downto_sample is hard-coded to 0.1 rather than using the configuration value, in two places within network_control.py. 

Addendum: changing 
```
tr.downsample_to(RestDownconf.deltat_down(0.1)
```
to

```
tr.downsample_to(RestDownconf.deltat_down,allow_upsample_max=3,snap=True)
```
seems to have solved the issue (the other parameters seemed a good idea to me after reading downsample_to documentation. [ Would send a PR but on ship internet and not so familiar with this procedure I don't have the patience now ]

**Comment by gesape** [comment]
The part where downsampling to 0.1 is applied is not the "final downsampling" to the sampling rate that is set in the config file. The intermediate downsampling to 0.1 was only introduced for an easier handling of the restitution of the data. The "final downsampling to the required rate is applied in the function downsample_rotate at lines 1175, 1200 and 1220 for the three channels.

You are however absolutely correct that the previous intermediate downsampling step can fail if fixed to 0.1, and is also not needed if the original sampling rate is already low.
I will fix that now and push an update. Thanks for the feedback.

**Comment by gesape** [comment]
I pushed an update that I tested with GR array stations, LH and HH components. Let me know if you run into any other problems.
